src/main/java/util/UserAuthUtil.java

```diff
@@ -0,0 +1,56 @@
+package util;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+public class UserAuthUtil {
+  /**
+   * Returns a boolean for the user's login status
+   * @return user login status
```

Comment:
For my comments, I haven't been doing @param or @return. Is this mentioned in the java style guide? I'm ok with it, everybody just needs to be on the same page for their comments then.

Reply:
I think I just used regular comments rather than @param or @return... but I think it may be up to preference or the style guide.

Reply:
These are javadoc comments to help with documenting the API. You can read about it [here](https://google.github.io/styleguide/javaguide.html#s7-javadoc), but let's talk about it today during standup?

Reply:
+1 for javadoc-style @param and @return

Reply:
An extra nit about javadocs: end sentences with periods.